client/templates/dashboard/dashboard.js

```diff
@@ -1,3 +1,7 @@
+Template.dashboardShopifyProducts.onCreated(function () {
+  this.subscribe('ShopifyProducts/CurrentImport');
+});
+
```

Comment:
I think the new best practice for this is to subscribe on a template rather than in the Router. Flow Router does away with router subscriptions all together.